collector/src/runtime/benchmark.rs

```diff
@@ -0,0 +1,104 @@
+use benchlib::benchmark::passes_filter;
+use cargo_metadata::Message;
+use core::option::Option;
+use core::option::Option::Some;
+use core::result::Result::Ok;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+/// A binary that defines several benchmarks using the `benchmark_suite` function from `benchlib`.
+#[derive(Debug)]
+pub struct BenchmarkSuite {
```

Comment:
Ok, I think I have a coherent set of definitions I'd like us to use.
- "benchmark suite" refers to all the benchmarks. So rustc-perf has a compile-time benchmark suite and a runtime benchmark suite.
- "benchmark" refers to a single thing that you can run (possibly more than once) and measure. 
  - For compile-time benchmarking, this is a single crate that gets compiled by rustc.
  - For runtime benchmarking, this is a single function that gets compiled by rustc and then run.
- "benchmark group" is the intermediate level. (Criterion also uses "group".) Each separate binary defines a group. The benchmark suite will contain multiple binaries (there are already three) and each benchmark binary will contain one or more benchmarks, which should be closely related.

So this type should be called `BenchmarkGroup`.